myproject/blog/tests.py

```diff
@@ -1,3 +1,117 @@
 from django.test import TestCase
+from .models import Author, Comment, BlogPost
+from django.contrib.auth.models import User
+from django.urls import reverse
 
-# Create your tests here.
+class ModelsObjectNameTests(TestCase):
+    
+    def test_models_has_expected_obj_name(self):
+        test_user = User.objects.create(username="test_user",password="test878")
+        author = Author.objects.create(name="Test Author",user=test_user,biological_info="test info")
+        blog = BlogPost.objects.create(title="Test Title",author = author, content= "This is content")
+        comment = Comment.objects.create(user=test_user,blog=blog,comment="This is comment" )
+        self.assertEqual(str(blog),"Test Title")
+        self.assertEqual(str(comment),"This is comment")
+        self.assertEqual(str(author), "Test Author")
+        
+class AuthorModelsFieldLabel(TestCase):
+    
+    @classmethod
+    def setUpTestData(cls):
+        test_user1 = User.objects.create_user(username='testuser1', password='12345') 
+        test_user1.save()
+        Author.objects.create(user=test_user1,name="test_author",biological_info='This is a bio')
+        
+    def test_user_label(self):
+        author=Author.objects.get(id=1)
+        field_label = author._meta.get_field('user').verbose_name
+        self.assertEquals(field_label,'user')
+        
+    def test_name_label(self):
+        author=Author.objects.get(id=1)
+        field_label = author._meta.get_field('name').verbose_name
+        self.assertEquals(field_label,'name')
+        
+    def test_biological_label(self):
+        author=Author.objects.get(id=1)
+        field_label = author._meta.get_field('biological_info').verbose_name
+        self.assertEquals(field_label,'biological info')
```

Comment:
Explain me here what are you testing?

Reply:
All model fields have the correct label.

Reply:
We don't want to test that. We want to test if the name of the author is correct or any other information related to the author. You are checking if the field exist or not in the database table which is useless and not useful.